cms/djangoapps/contentstore/rest_api/v2/urls.py

```diff
@@ -29,6 +29,11 @@
         downstreams.UpstreamListView.as_view(),
         name='upstream-list'
     ),
+    re_path(
+        f'^upstream/{settings.USAGE_KEY_PATTERN}/downstream-contexts$',
```

Comment:
I'm not happy with my choice of URL here. I was trying to have a structure like parent/{parentId}/children instead of children/{parentId}, but I`m not sure if this is clear close to the other endpoints.

Do you think there is a better way to make it consistent with the other URLs?